### PR TITLE
Adjustments to the Blue-Green Deployment topic

### PR DESCRIPTION
**Pull request by ahardin-rh**
Minor edits per discussion with @sdodson 

- Moved the graphic down to the end of its section
- Fixed a typo 
- Added a link to canary deployments discussion
- Added an example

### CODE REVIEW

File: install_config/upgrading/blue_green_deployments.adoc [hunk 1]
```diff
@@ -175,6 +175,17 @@ xref:../../architecture/infrastructure_components/kubernetes_infrastructure.adoc
 (which you will verify in later steps), set the `openshift_schedulable=false`
 variable for each node host to ensure they are unschedulable initially.
 
+.Example of Adding New Nodes
```
Comment (sdodson) [inline install_config/upgrading/blue_green_deployments.adoc]: Maybe the title here should be `Example *new_nodes* Host Group` ?

We should make a note here that the example below should be added to their existing inventory, everything that was in there previously should remain.

**Comment by ahardin-rh** [comment]
@sdodson Updated. 

@openshift/ansible-design Please peer review

File: install_config/upgrading/blue_green_deployments.adoc [hunk 2]
```diff
@@ -232,8 +246,9 @@ projects should be isolated and pods in global projects should be able to access
 == Registry and Router Canary Deployments
 
 A common practice is to scale the registry and router pods until they are
-migrated to new (green) infrastructure node hosts. For these pods, a _canary_
-deployment approach is commonly used.
+migrated to new (green) infrastructure node hosts. For these pods, a
+xref:../../dev_guide/deployments/deployment_strategies.adoc#canary-deployments.adoc#canary-deployments[_canary_
```
Comment (adellape) [inline install_config/upgrading/blue_green_deployments.adoc]: Double anchors / .adocs!

**Review by adellape — COMMENTED** [review]
One comment, otherwise LGTM.

File: install_config/upgrading/blue_green_deployments.adoc (hunk 2 above)
Comment (ahardin-rh) [inline install_config/upgrading/blue_green_deployments.adoc]: :scream_cat: Whoops! Thanks @adellape

**Comment by ahardin-rh** [comment]
@adellape Thanks for the review. Fixed. Mergin' now!

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../install_config/upgrading/blue_green_deployments.adoc#upgrading-blue-green-deployments[Upgrading a Cluster -> Blue-Green Deployments]
|Added an example `new_nodes` host group to xthe ref:../install_config/upgrading/blue_green_deployments.adoc#blue-green-creating-labeling-green-nodes[Creating and Labeling Green Nodes] section.
%

**Pull request merged**